src/Menu/Menu.js

```diff
@@ -23,11 +23,17 @@ const StyledDiv = Div.extend`
   svg {
     fill: currentColor;
   }
+  .top {
+    justify-content: flex-end;
+    svg {
+      margin-right: 1.5rem;
+    }
+  }
   .bottom {
     &.small {
       display: none;
     }
-    height: 4rem;
+    height: calc(4rem + 1px);
```

Comment:
This aligns menu items text with nav items text.
Menu items are smaller and have text on the right side of the icon, nav items have at the bottom.